3494.04 <c-53#17> Created logger
3494.21 <c-53#17:56> Logger was updated
3494.21 <c-53#17:56> Started intializing cube lib
3494.21 <c-53#17:56> Finished intializing cube lib with worker
3495.74 <c-53#17:56> shareCubes is called
3495.74 <c-53#17:56> Added 0 failed local cubes
3495.74 <c-53#17:56> Pruned 0 local cubes
3495.74 <c-53#17:56> CubeSolverThread entered cube retrieval loop, currently there are 0 cubes
3495.74 <c-53#17:56> returnFailedAndRequestCubes => [48] {}
3495.74 <c-53#17:56> Sent 0 failed cubes to root and requested new cubes
3495.74 <c-53#17:56> Received 4 cubes from root
3495.74 <c-53#17:56> CubeSolverThread entered cube retrieval loop, currently there are 4 cubes
3495.74 <c-53#17:56> CubeSolverThread retrieved 4 cubes
3495.74 <c-53#17:56> Started solving a cube
3495.74 <c-53#17:56> Cube failed
3495.74 <c-53#17:56> Added failed cube
3495.74 <c-53#17:56> Started solving a cube
3551.49 <c-53#17:56> appl_pause was called
3551.67 <c-53#17:56> SUSPEND (0.46s since last cb)
4002.15 <c-53#17:56> appl_interrupt was called
4002.15 <c-53#17:56> RESUME
4002.15 <c-53#17:56> STOP after suspension
4002.15 <c-53#17:56> Solving interrupted
4002.15 <c-53#17:56> Leaving the main loop
4002.15 <c-53#17:56> appl_interrupt was called
4002.15 <c-53#17:56> appl_withdraw was called
4002.21 <c-53#17:56> Started withdraw thread
4002.78 <c-53#17:56> Enter destructor of CubeWorkerInterface
4002.78 <c-53#17:56> Enter destructor of CubeRoot
4002.91 <c-53#17:56> Finished withdraw thread
4063.48 <c-53#17:56> Enter destructor
4063.48 <c-53#17:56> Time working: 57.280
4063.48 <c-53#17:56> Time suspended: 450.658
4063.48 <c-53#17:56> Joined cleanup thread
4063.48 <c-53#17:56> Exit destructor
4063.48 <c-53#17:56> Destructing logger
